src/icons/badges/MetadataActiveBadge.js

```diff
@@ -23,7 +23,7 @@ const MetadataActiveBadge = ({ className = '', height = 16, title, width = 16 }:
             <circle cx="8" cy="8" fill="none" r="7.5" stroke="#4D91E2" />
```

Comment:
#4D91E2 too

Reply:
I believe this should be $bdl-box-blue instead of #4D91E2, will make the change for that

Reply:
Hm, this color doesn't appear to be one of our main hues. can you attach a screenshot of the affected icons?

Reply:
Ah, from the icon this should be box blue. good catch